dataframe-excel/src/main/kotlin/org/jetbrains/kotlinx/dataframe/io/xlsx.kt

```diff
@@ -83,6 +84,9 @@ private fun setWorkbookTempDirectory() {
 /**
  * @param sheetName sheet to read. By default, the first sheet in the document
  * @param columns comma separated list of Excel column letters and column ranges (e.g. “A:E” or “A,C,E:F”)
+ * @param formattingOptions range of columns to read as String regardless of a cell type.
```

Comment:
It's not just that right? You can format a range of columns as string using different options as defined by a `DataFormatter`. The description should probably reflect this.

If this argument was just to format columns as string, the argument should be `colRangeAsString` or better, mirror the `colTypes` argument from CSV.

Reply:
Changed the parameter for some overloads to StringColumns class and provided see also where it's still FormattingOptions